OpenSim/Examples/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -174,22 +174,16 @@ int main(int argc, char* argv[]) {
     // Following are arguments to this executable:
     // (1) noVisualizer -- [Optional] Do not show visualizer. Default
     //                       behavior is to show visualizer.
-    // (2) simulateOnce -- [Optional] Simulate once and exit the program.
```

Comment:
Suggest cleaning up "Following are arguments to this executable:" list since there is now only 1 argument.